src/Microsoft.AspNetCore.Mvc.Core/Internal/ApiBehaviorApplicationModelProvider.cs

```diff
@@ -273,6 +274,21 @@ internal static void DiscoverApiConvention(ActionModel actionModel, ApiConventio
             }
         }
 
+        internal void DiscoverApiErrorType(ActionModel actionModel)
+        {
+            if (_apiBehaviorOptions.SuppressUseClientErrorFactory)
+            {
+                return;
+            }
```

Comment:
Why should the behavior of this be tied to `SuppressUseClientErrorFactory`?

Reply:
So I had another stab at this - we pivot on this value to determine if we need to set a default. Seems kinda excessive to have a separate property to disable the default